extensions/amp-social-share/1.0/social-share.type.js

```diff
@@ -18,7 +18,7 @@
 
 /**
  * @typedef {{
- *   type: (string|undefined),
+ *   type: (!string),
```

Comment:
@dvoytenko I updated this to non-nullable string.  Is this what you meant by `strict string`?

Reply:
Strict string is just `string`. I.e. no `?` or `|null` or `|undefined` or any other `|`. The `!` is not needed for primitive types, and I think the Closure styleguide complains about it.

Reply:
Got it, thanks for clarifying!